### DISCUSSION TITLE
Merged Subtitles Alignment

### DISCUSSION THREAD

**Discussion by lohcm88** [SubtitleEdit/subtitleedit#5871] [Q&A] [answered]
Under right click " ALIGNMENT " , there are only 9 options for subtitles placement, is there a way to place 2 merged subtitles just slightly above each other rather than only BOTTOM/CENTER, MIDDLE/CENTER or TOP/CENTER etc

![Capture](https://user-images.githubusercontent.com/18275197/160556289-00c18d25-1273-4129-9fde-40e5ba4d2add.JPG)

**Comment by niksedk** [comment, marked as answer]
If you use subtitle format "Advanced Sub Station Alpha", you can use any alignment you want, but only one alignment for each line (but lines can overlap or have same timestamps)
